src/main/resources/application.properties

```diff
@@ -48,3 +48,41 @@ app.table.detectedLicenseSchema=lpvs
 app.table.diffFileName=code_licenses
 app.table.pullRequestsName=lpvs_pull_requests
 app.table.queueName=queue
+
+# Login Configuration
+
+#Google
+spring.security.oauth2.client.registration.google.client-id=
+spring.security.oauth2.client.registration.google.client-secret=
+spring.security.oauth2.client.registration.google.redirect-uri=http://localhost:7896/login/oauth2/code/google
+spring.security.oauth2.client.registration.google.scope=profile, email
+
+#Naver
+spring.security.oauth2.client.registration.naver.client-id=
+spring.security.oauth2.client.registration.naver.client-secret=
+
+spring.security.oauth2.client.registration.naver.redirect-uri=http://localhost:7896/login/oauth2/code/naver
+spring.security.oauth2.client.registration.naver.authorization-grant-type=authorization_code
+spring.security.oauth2.client.registration.naver.scope=name, email, profile_image
+spring.security.oauth2.client.registration.naver.client-name=Naver
+#Provider-Naver
+spring.security.oauth2.client.provider.naver.authorization-uri=https://nid.naver.com/oauth2.0/authorize
+spring.security.oauth2.client.provider.naver.token-uri=https://nid.naver.com/oauth2.0/token
+spring.security.oauth2.client.provider.naver.user-info-uri=https://openapi.naver.com/v1/nid/me
+spring.security.oauth2.client.provider.naver.user-name-attribute=response
+
+#Kakao
+spring.security.oauth2.client.registration.kakao.client-id=
+spring.security.oauth2.client.registration.kakao.redirect-uri=http://localhost:7896/login/oauth2/code/kakao
+spring.security.oauth2.client.registration.kakao.client-authentication-method=POST
+spring.security.oauth2.client.registration.kakao.authorization-grant-type = authorization_code
+spring.security.oauth2.client.registration.kakao.scope=profile_nickname, profile_image, account_email
+spring.security.oauth2.client.registration.kakao.client-name=Kakao
+#Provider-Kakao
+spring.security.oauth2.client.provider.kakao.authorization-uri=https://kauth.kakao.com/oauth/authorize
+spring.security.oauth2.client.provider.kakao.token-uri=https://kauth.kakao.com/oauth/token
+spring.security.oauth2.client.provider.kakao.user-info-uri=https://kapi.kakao.com/v2/user/me
+spring.security.oauth2.client.provider.kakao.user-name-attribute=id
```

Comment:
It would be better leave these lines of code as comments since login function is not default.

Reply:
As an alternative, I put temporary value to modify if user want.